app/src/shared/components/Nav/index.jsx

```diff
@@ -75,21 +75,28 @@ const Nav = compose(
                 label={I18n.t('general.docs')}
                 to={routes.docs()}
             >
-                <Link to={routes.docsGettingStarted()}>
-                    <Translate value="general.gettingStarted" />
-                </Link>
-                <Link to={routes.docsStreamsRoot()}>
-                    <Translate value="general.streams" />
-                </Link>
-                <Link to={routes.docsCanvasesRoot()}>
-                    <Translate value="general.canvases" />
-                </Link>
-                <Link to={routes.docsDashboards()}>
-                    <Translate value="general.dashboards" />
-                </Link>
-                <Link to={routes.docsProductsRoot()}>
-                    <Translate value="general.products" />
-                </Link>
+                {[
```

Comment:
Needed to change to a list because simply adding something between the `Links` which is not shown makes a separator:

```js
<Link to={routes.docsDashboards()}>...</Link>
{condition ? ...}
<Link to={routes.docsDashboards()}>...</Link>
```